src/components/Rockets.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux/es/hooks/useSelector';
+import { fetchRockets } from '../redux/rocket/rocketSlice';
+import Rocket from './Rocket';
+
+function Rockets() {
+  const rocketItems = useSelector((state) => state.rocket.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+  return (
+    <div className="container">
+      {rocketItems.map((rocket) => (
+        <Rocket
+          key={rocket.id}
+          id={rocket.id}
+          image={rocket.image}
+          name={rocket.rocket_name}
+          description={rocket.description}
+          reserved={rocket.reserved}
+        />
+      ))}
+    </div>
+  );
+}
+
+export default Rockets;
```

Comment:
- Optional
The rocket reservation data does not persist when I navigate to the profile and back. Since the previous reviewer didn't raise this as required, I am marking this as optional. However, I would recommend you implement this. It would be a good challenge for debugging and learning more about redux.